30 paskaita (egzaminas)/Task 4/style.css

```
@@ -27,3 +28,32 @@
   margin-top: 50px;
   color: #333;
 }
+
+#output {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+}
+
+.car-card {
+  border: solid black 1px;
+  border-radius: 5px;
+  width: 250px;
+  margin: 1rem;
+  padding: 10px;
+  background-color: rgb(175, 206, 203);
+}
+
+.brand {
+  text-align: center;
+  font-size: 20px;
+  margin-bottom: 1rem;
+}
+
+li {
+  list-style: none;
+}
+
+ul>li:nth-child(odd) {
+  background-color:rgb(143, 201, 196)
```

Geras budas paryskinti kas antra eilute 👍 

Nitpick: formatavimas
```suggestion
  background-color: rgb(143, 201, 196);
```